### DISCUSSION TITLE
Why doesn't the project I created have files from multiple platforms?

### DISCUSSION THREAD

**Discussion by bbhxwl** [AvaloniaUI/Avalonia#15943] [General]
Why doesn't the project I created have files from multiple platforms?

![image](https://github.com/AvaloniaUI/Avalonia/assets/26525599/a822c2d4-1a5b-4ee0-840a-5d1fa6b3920c)

**Comment by rabbitism** [comment]
What is your expectation?

**Reply by bbhxwl** [reply]
Perhaps there is an issue with the project I created. Thank you very much.
